sdcm/gemini_thread.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import concurrent.futures
+import os
+import uuid
+import random
+import json
+import re
+import time
+
+from sdcm.sct_events import GeminiToolEvent, GeminiToolLogEvent, Severity
+from sdcm.utils import FileFollowerThread
+
+LOGGER = logging.getLogger(__name__)
+
+
+class GeminiToolEventsPublisher(FileFollowerThread):
+    severity_mapping = {
+        "INFO": "NORMAL",
+        "DEBUG": "NORMAL",
+        "WARN": "WARNING",
+        "ERROR": "ERROR",
+        "FATAL": "CRITICAL"
+    }
+
+    def __init__(self, node, gemini_log_filename):
+        super(GeminiToolEventsPublisher, self).__init__()
+        self.gemini_log_filename = gemini_log_filename
+        self.node = str(node)
+        self.gemini_events = [GeminiToolLogEvent(type='geminievent', regex=r'{"L":"(?P<type>[A-Z]+?)".+"M":"(?P<message>[\w\s.,]+?)"}', severity=Severity.CRITICAL)]
```

Comment:
if `GeminiToolEvent` changes to `GeminiToolEvent`
so `GeminiToolLogEvent` should also change to `GeminiLogEvent`

Reply:
renamed